include/gbinder_local_object.h

```diff
@@ -61,6 +61,15 @@ GBinderLocalReply*
 gbinder_local_object_new_reply(
     GBinderLocalObject* obj);
 
+GBINDER_STABILITY_LEVEL
+gbinder_local_object_get_stability(
+    GBinderLocalObject* self); /* Since 1.0.40 */
```

Comment:
Is there a good  reason (or any reason at all) to make this function external? If not, then there's no need to have this function at all, even as an internal one - since  `gbinder_rpc_protocol.c` needs to include `gbinder_local_object_p.h` anyway and then it can just directly access the `stability` field (checking the object pointer for `NULL` first). I don't like exposing unnecessary API because unnecessary things have a tendency to pile up and become a burden from the backward compatibility POV.

In other words, I suggest to get rid of it))

Reply:
You're right, I don't think it's needed in the external code. Just wanted it to be symmetric with set/get, but can get rid of it then.